src/coins.cpp

```diff
@@ -326,8 +328,8 @@ void CCoinsViewCache::SanityCheck() const
     size_t recomputed_usage = 0;
     for (const auto& [_, entry] : cacheCoins) {
         unsigned attr = 0;
-        if (entry.flags & CCoinsCacheEntry::DIRTY) attr |= 1;
-        if (entry.flags & CCoinsCacheEntry::FRESH) attr |= 2;
+        if (entry.GetFlags() & CCoinsCacheEntry::DIRTY) attr |= 1;
+        if (entry.GetFlags() & CCoinsCacheEntry::FRESH) attr |= 2;
```

Comment:
```suggestion
        auto attr = entry.GetFlags();
```

(or just remove the assert, doesn't make a lot of sense to me)

Reply:
Not sure I want to touch this right now. C